pandas/core/resample.py

```diff
@@ -1501,6 +1501,8 @@ def _get_time_delta_bins(self, ax):
         # Addresses GH #10530
         if self.base > 0:
             labels += type(self.freq)(self.base)
+        if self.loffset:
+            labels += self.loffset
```

Comment:
Useful because otherwise `tests/resample/test_base.py::test_resample_loffset_arg_type` was always failing. However, this code is questionable since #31809 will deprecate `loffset` anyway.

Reply:
can you add a GH ref comment here